### ISSUE TITLE
browse everything config not deployed in production

### ISSUE THREAD

**Issue by bess** [curationexperts/laevigata#751]
Error in `/opt/laevigata/current/logs/production.log`:
```
F, [2017-09-12T04:00:21.287184 #23193] FATAL -- : [55c7abf7-b7ce-4b18-a74d-29d41fdc724d] ActionView::Template::Error (No such file or directory @ rb_sysopen - /opt/laevigata/releases/20170824002358/config/browse_everything_providers.yml):
```
Checking `/opt/laevigata/current/config`, indeed the expected `browse_everything_providers.yml` file is not present.

**Comment by mark-dce** [comment]
Which environment is this happening on?  (QA / STAGE / PROD)

The file needs to be absent to disable the browse everything sections on the ETD upload form from rendering.

Capistrano creates a link to the file unless the `config/deploy.rb` file is updated - see https://github.com/curationexperts/laevigata/commit/d873586609736191c7168ecb57c9b844642b6199#diff-6dc906be0643182e5bae6aa1506221e6

So, the two changes need to be coordinated, either:
ENABLE: have the sharing line in `config/deploy.rb` and place the proper config file in`/opt/laevigata/shared/config` on the server
OR
DISABLE: comment out the line in 'config/deploy.rb` and ensure the config file is not checked into the repo

**Comment by acozine** [comment]
The config is in the correct place on all 3 apache servers, and the repo only has a `config/browse_everything_providers.yml.sample`, so when we're ready to roll out the Box functionality, all we need to do is change `config/deploy.rb` and deploy.

**Comment by acozine** [comment]
Closed by 27bba9f1c7daeee9c0b647b65b611f3b3186c3bb
